o2.php

```diff
@@ -1161,7 +1163,13 @@ public static function is_mobile() {
 	 * Returns true if the user agent is recognized as a tablet
 	 */
 	public static function is_tablet() {
-		$is_tablet = ( class_exists( 'Jetpack_User_Agent_Info' ) ) ? Jetpack_User_Agent_Info::is_tablet() : false;
+		if ( class_exists( '\Automattic\Jetpack\Device_Detection\User_Agent_Info' ) ) {
+			$is_tablet = ( new User_Agent_Info )->is_tablet();
```

Comment:
I don't see a minimum PHP or WP version declared for this, so not sure if we can safely use PHP 5.4 syntax.

Reply:
Done in e36bbb1